label_studio/tasks/models.py

```diff
@@ -992,6 +992,44 @@ class Meta:
         db_table = 'prediction'
 
 
+class FailedPrediction(models.Model):
+    """
+    Class for storing failed prediction(s) for a task
+    """
+    message = models.TextField(
+        _('message'),
+        default='',
+        blank=True,
+        null=True,
+        help_text='The message explaining why generating this prediction failed'
+    )
+    error_type = models.CharField(
+        _('error_type'),
+        max_length=512,
+        default=None,
+        null=True,
+        help_text='The type of error that caused prediction to fail'
+    )
+    model_version = models.TextField(
```

Comment:
Yes that is true, however this field will be used in case we are using a ML backend as well. Reference to the model itself + model version (just like we do for predictions)